### ISSUE TITLE
Hydraulics import

### ISSUE THREAD

**Issue by rnugent3** [HydrologicEngineeringCenter/HEC-FDA#442]
Please write the logic to import hydraulics from HDF files for both steady and unsteady. 

Please speak with @Brennan1994 about the details.

**Comment by Brennan1994** [comment]
I would like to amend this language to say, please write the controls and VMs to import steady and unsteady. There will need to be a new VM  and Control for each of these. We should still discuss the details when you're ready @codymccoy

**Comment by codymccoy** [comment]
My notes from a meeting with Brennan:
![image](https://user-images.githubusercontent.com/47757848/180006021-a3084cfb-f296-4460-8c0d-0689e5174e60.png)

**Comment by codymccoy** [comment]
This task is mostly done. I am waiting on Brennan to fix a file/folder browser bug where it is setting the path twice. I also am waiting for my latest change to get merged into framework where the file browser checks if the ok was clicked before setting the path. Also waiting for Brennan to figure out how to use RasMapper to get the list of strings for "Steady". Right now i am using a dummy list of strings.

**Comment by rnugent3** [comment]
It looks like we're looking for a single .hdf when attempting to import unsteady hydraulics. Instead, we should be looking for a _folder_ that contains 8 .hdfs

**Comment by rnugent3** [comment]
looks like the dummy list of strings is cody kayak and test, right?

**Comment by codymccoy** [comment]
yes
